docs/upgrading.md

```diff
@@ -5,6 +5,16 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.8.0` to `v0.9.0`
+
+Breaking changes
+
+- the `max_participants` have been removed. 
```

Comment:
removed where? This needs to be a full sentence that's clearly stating where the change occured